### PR TITLE
[egs] Improve aishell/s5/run.sh commentary

### PR DESCRIPTION
**Pull request by HuangruiChu**
For the monophone training, I think it only uses Delta. While for tri1 and tri2, Delta+Delta-Delta  (first triphone pass) is used. I also make it clear what is [LDA+MLLT (second triphone pass)] and [LDA+MLLT+SAT (third triphone pass)]

### CODE REVIEW

File: egs/aishell/s5/run.sh [hunk 1]
```diff
@@ -87,14 +88,15 @@ steps/decode.sh --cmd "$decode_cmd" --config conf/decode.config --nj 10 \
 steps/decode.sh --cmd "$decode_cmd" --config conf/decode.config --nj 10 \
   exp/tri2/graph data/test exp/tri2/decode_test
 
-# train and decode tri2 [delta+delta-deltas]
+# align tri2 [delta+delta-deltas]
```
Comment (kkm000) [inline egs/aishell/s5/run.sh:91]:
```suggestion
# Align training data with the tri2 model.
```
(move the feature type to the point where the tri2 model is trained)

File: egs/aishell/s5/run.sh [hunk 2]
```diff
@@ -87,14 +88,15 @@ steps/decode.sh --cmd "$decode_cmd" --config conf/decode.config --nj 10 \
...
 steps/align_si.sh --cmd "$train_cmd" --nj 10 \
   data/train data/lang exp/tri2 exp/tri2_ali || exit 1;
 
-# train tri3a, which is LDA+MLLT,
+# train tri3a [LDA+MLLT (second triphone pass)]
```
Comment (kkm000) [inline egs/aishell/s5/run.sh:95]:
```suggestion
# Train the second triphone pass model tri3a on LDA+MLLT features.
```

File: egs/aishell/s5/run.sh [hunk 3]
```diff
@@ -87,14 +88,15 @@ steps/decode.sh --cmd "$decode_cmd" --config conf/decode.config --nj 10 \
...
 steps/train_lda_mllt.sh --cmd "$train_cmd" \
  2500 20000 data/train data/lang exp/tri2_ali exp/tri3a || exit 1;
-
+ 
+# decode tri3a
```
Comment (kkm000) [inline egs/aishell/s5/run.sh:99]:
```suggestion
# Run a test decode with the tri3a model.
```

File: egs/aishell/s5/run.sh [hunk 4]
```diff
@@ -106,8 +108,8 @@ steps/decode.sh --cmd "$decode_cmd" --nj 10 --config conf/decode.config \
 steps/align_fmllr.sh --cmd "$train_cmd" --nj 10 \
   data/train data/lang exp/tri3a exp/tri3a_ali || exit 1;
 
-# train tri4a, which is LDA+MLLT+SAT
-# From now, we start building a more serious system (with SAT)
+# train tri4a [LDA+MLLT+SAT (third triphone pass)]
+# From now, we start building a more serious system with Speaker Adaptive Training(SAT)
```
Comment (kkm000) [inline egs/aishell/s5/run.sh:112]:
```suggestion
# Train the third triphone pass model tri4a on LDA+MLLT+SAT features.
# From now on, we start building a more serious system with Speaker
# Adaptive Training (SAT).```

**Review by kkm000 — CHANGES_REQUESTED** [review]
Thank you for improving our commentary! It's really a lot of dirty work, but does support our general code quality, and is really appreciated. Speaking of scripts, and assuming you want to contribute more to the spring cleanup, I think that the commentary that needs the most attention is in the two common directory subtrees `egs/wsj/s5/{utils,steps}`. Individual recipes—all other files under `egs/`, essentially—are too numerous to go through systematically, and are not read as often. A `run.sh` is always a starting point for someone starting to dig into a recipe; the problem is that there are currently 126 of them, and none can be called more exemplary than any other. Those in `egs/wsj/s5/` are too not special, except for these two special subdirectories: this is just a historical layout artifact.

Having said that, the [Google Style guidelines](https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts) (read "Don'ts" and "Punctuation, Spelling, and Grammar" subsections, they apply to all languages) call for comments that are complete sentences, and are capitalized and punctuated as such. I admit that this rule is violated more often than completely observed in our codebase. Your PR in indeed welcome and acceptable, but if improving comments, let's do it right. I'm adding a proposed change where the coding standard is not followed, just to give an example that follows the Guide to a comma. You can just accept them in GitHub interface if you agree. General points for commenting:

 * Write a full sentence, starting with a capital letter and ending with a period. Imperative mood ("Train a model") is the most concise and preferred for a one-sentence comment: code _commands_ the computer to do things, and a comment explains the command, but it's still a command. This is not hard-and-fast rule, just a recommendation; context is important.
 * Add one space between a comment character and text.
 * Wrap lines at 70-80 characters,  exceed 80, and remove end-of-line space.

**Comment by jtrmal** [comment]
@HuangruiChu I wonder if you could find time to address these comments?

**Comment by kkm000** [comment]
LGTM, and thanks! If you ever want to look into improving comments, the common scripts are of a higher priority than individual egs.

**Review by kkm000 — APPROVED** [review]

**Pull request merged**